docs/toc.yml

```
@@ -149,26 +149,40 @@
           href: design/data-visualization-guidelines.md
         - name: Voice
           href: design/voice-guidelines.md
-      - name: Design language
+      - name: UI design language and patterns
```

Since some of the contents include Authentication and First Run XP, should this be labelled "UX" instead of "UI"?
```suggestion
      - name: UX design language and patterns
```